### DISCUSSION TITLE
Batch Script Clarification

### DISCUSSION THREAD

**Discussion by GreenPhenz** [smacke/subsync#185] [Q&A]
I've seen the post on how to use this sricpt for batches:
`for i in {1..10}; do
    ffs reference/$i.mkv -i in/$i.srt -o out/$i-synced.srt
done`
But I don't understand where the input files are supposed to go. What is the reference/, in/, and out/ supposed to be replaced with? Also, is the output file supposed to already be created or can the program automatically make one (and if so how can I direct which folder for it to go in?)?

Also, I'm on mac if that changes anything

**Comment by DJHanceNL** [comment]
for %i in (*.mp4) do ffs %~ni.mp4 -i %~ni.srt -o %~ni_sync.srt
with clean file names

with spaces or shit
for %i in (*.mp4) do ffs "%~ni.mp4" -i "%~ni.srt" -o "%~ni_sync.srt"

in terminal

or change mp4 to .mkv.

now you only got to remove the bad srt and replace with the _sync ones
